tests/calc/test_thermo.py

```diff
@@ -22,6 +22,7 @@
                         relative_humidity_from_mixing_ratio,
                         relative_humidity_from_specific_humidity,
                         relative_humidity_wet_psychrometric,
+                        richardson_number,
```

Comment:
```suggestion
                        gradient_richardson_number,
```
(note: will need to change import order once this is changed to make flake8 happy.)